front/app/containers/Admin/settings/approval/index.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect, useState } from 'react';
+
+import {
+  IOption,
+  Button,
+  Box,
+  Spinner,
+} from '@citizenlab/cl2-component-library';
+
+import useUpdateUser from 'api/users/useUpdateUser';
+import useUsers from 'api/users/useUsers';
+
+import {
+  Section,
+  SectionDescription,
+  SectionField,
+  SectionTitle,
+} from 'components/admin/Section';
+import MultipleSelect from 'components/UI/MultipleSelect';
+
+import { useIntl } from 'utils/cl-intl';
+import { getFullName } from 'utils/textUtils';
+
+import messages from './messages';
+
+const Approval = () => {
+  const { formatMessage } = useIntl();
+
+  const [selectedUsers, setSelectedUsers] = useState<IOption[]>([]);
+  const { mutate: updateUser, isLoading } = useUpdateUser();
+  const { data: adminUsers } = useUsers({
+    can_admin: true,
+  });
+  const { data: projectReviewers } = useUsers({
+    project_reviewer: true,
+  });
+
+  useEffect(() => {
+    if (projectReviewers) {
+      setSelectedUsers(
+        projectReviewers.data.map((user) => ({
+          value: user.id,
+          label: getFullName(user),
+        }))
+      );
+    }
+  }, [projectReviewers]);
```

Comment:
Just for my own understanding, would something like this work?

```suggestion
  useEffect(() => {
    const reviewers = projectReviewers?.data || [];

    setSelectedUsers(
      reviewers.map((user) => ({
        value: user.id,
        label: getFullName(user),
      }))
    );
  }, [projectReviewers]);
```